integration-libs/s4-service/order/core/facade/reschedule-service-order.service.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, Injectable } from '@angular/core';
+import { RescheduleServiceOrderConnector } from '../connector';
+import {
+  RescheduleServiceOrderFacade,
+  ServiceDateTime,
+} from '@spartacus/s4-service/root';
+import { UserIdService } from '@spartacus/core';
+import { switchMap } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RescheduleServiceOrderService
+  implements RescheduleServiceOrderFacade
+{
+  protected rescheduleServiceOrderConnector = inject(
+    RescheduleServiceOrderConnector
+  );
+  protected userIdService = inject(UserIdService);
+
+  constructor() {}
+
+  rescheduleService(orderCode: string, scheduledAt: ServiceDateTime) {
```

Comment:
Check if return type observable need to be added?

Reply:
Done.